From: Dan Streetman <ddstreet@canonical.com>
Bug: https://github.com/martinpitt/umockdev/issues/105
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/umockdev/+bug/1831467

This assert isn't needed for operation; nothing is done with the read byte.
However it intermittently causes autopkgtest failures, so just ignore it.
A bug is open upstream to track investigating why it's failing.

--- a/src/umockdev.vala
+++ b/src/umockdev.vala
@@ -1670,7 +1670,7 @@ private class SocketServer {
             if (Posix.FD_ISSET (this.ctrl_r, fds) > 0) {
                 debug ("socket server thread: woken up by control fd");
                 char buf;
-                assert (Posix.read (this.ctrl_r, &buf, 1) == 1);
+                Posix.read (this.ctrl_r, &buf, 1);
                 continue;
             }
 
